linera-chain/src/manager.rs

```diff
@@ -158,8 +158,8 @@ where
     /// proposals in the same round, this contains only the first one.
     #[graphql(skip)]
     pub proposed: RegisterView<C, Option<BlockProposal>>,
-    /// Latest validated proposal that we have voted to confirm (or would have, if we are not a
-    /// validator).
+    /// Latest validated proposal that a validator may have voted to confirm. This is either the
+    /// latest `ValidatedBlock` we have seen, or the proposal from the `Fast` round.
     #[graphql(skip)]
     pub locked: RegisterView<C, Option<LockedBlock>>,
```

Comment:
Maybe `externally_locked`, `validator_locked`, or `network_locked`?
Then, the container `LockedBlock` could be renamed accordingly.

Reply:
How would you feel about `UnlockingBlock`?

The point of it is that it can _unlock_ validators that have already voted to confirm.